### PR TITLE
upgrade to go 1.19

### LINKED ISSUE #672: Upgrade go version to 1.19
# Enhancement
Upgrade go version to 1.19

### PR DESCRIPTION
**Pull request by Leavrth**
Signed-off-by: Leavrth <jianjun.liao@outlook.com>

### What problem does this PR solve?

Issue Number: close #672 #599 

### What is changed and how it works?
upgrade to go 1.19

### CODE REVIEW

File: go.mod1 [hunk 1]
```diff
@@ -1,38 +1,209 @@
...
 	go.etcd.io/etcd/client/v3 v3.5.2
 	go.etcd.io/etcd/tests/v3 v3.5.2
 	go.uber.org/atomic v1.9.0
 	go.uber.org/zap v1.21.0
-	golang.org/x/net v0.0.0-20220516155154-20f960328961
-	google.golang.org/grpc v1.46.2
+	google.golang.org/grpc v1.48.0
+)
+
+require (
```
Comment (3pointer) [inline go.mod1:28]: 🤔

**Review by 3pointer — APPROVED** [review]

File: go.mod1 (hunk 1 above)
Comment (Leavrth) [reply go.mod1:28]: It seems that newer version of go mod split indirect and direct pkg.

**Pull request merged**